packages/core/nuxt-theme-module/README.md

```diff
@@ -0,0 +1,43 @@
+# Nuxt Theme Module
+This module:
+- Changes `.nuxt` buildDir to name you set in the config or `.theme` 
```

Comment:
it doesn't do this, `.nuxt` and `theme` are completely different things and they're not altering each other